docs/deployment/supported-proxies/caddy.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: default
+title: Caddy
+parent: Proxy Integration
+grand_parent: Deployment
+nav_order: 1
+---
+
+[Caddy] is a reverse proxy supported by **Authelia**. 
+
+_**Important:** Caddy officially supports the forward auth flow in version 2.5.1 and greater. You must be using this 
+version in order to use either Caddyfile. If you'd like to use it before 2.5.1 is released you need to build 
+[this branch](https://github.com/caddyserver/caddy/pull/4739)._ 
+
+Authelia offers integration support for the official forward auth integration method Caddy provides, we
+can't reasonably be expected to offer support for all of the different plugins that exist. As we have direct contact
+with the Caddy developers and their integration being written so well solving issues should be relatively straightforward.
```

Comment:
```suggestion
can't reasonably be expected to offer support for all of the different plugins that exist.
```

Not sure if we need this additional sentence, if you think we do probably something like this:

> As we have direct contact with the Caddy developers and their integration is written so well, solving issues should be relatively straightforward.